src/dialect/snowflake.rs

```diff
@@ -40,6 +43,8 @@ impl Dialect for SnowflakeDialect {
             || ch.is_ascii_digit()
             || ch == '$'
             || ch == '_'
+            || ch == '/'
```

Comment:
I didn't see any test coverage of these identifier changes. Could you please add some tests (to make sure we don't break the functionality in some future refactoring)?

Reply:
This is done within `test_snowflake_stage_object_names` test.